cli/src/command.rs

```diff
@@ -215,6 +205,21 @@ fn ensure_dev(spec: &Box<dyn service::ChainSpec>) -> std::result::Result<(), Str
 	}
 }
 
+/// Runs performance checks.
+/// Should only be run in release build since the check would take too much time otherwise.
+/// Returns `Ok` immediately if the check has been passed previously.
+fn host_perf_check(_result_cache_path: &Path, _force: bool) -> Result<()> {
+	#[cfg(not(build_type = "release"))]
+	{
+		Err(PerfCheckError::WrongBuildType.into())
+	}
+	#[cfg(build_type = "release")]
```

Comment:
will this work with custom profiles https://github.com/paritytech/polkadot/issues/4311#issuecomment-985084943 @bkchr?

Reply:
I don't think so, the profile should be matched with the one used for distributing the binary. If this profile changes to e.g `release-lto`, it should be also updated here.